hi-ml-azure/src/health_azure/utils.py

```diff
@@ -111,6 +112,20 @@
 V2_OUTPUT_DATASET_PATTERN = r"--OUTPUT_\d[=| ]"
 
 PathOrString = Union[Path, str]
+# An AML v1 Run or an AML v2 Job
+RunOrJob = Union[Run, Job]
+
+
+class JobStatus(Enum):
+    """String constants for the status of an AML v2 Job"""
+    Completed = "Completed"
```

Comment:
[Minor] Shouldn't Completed, Failed and Canceled be uppercase as they are constants?

Reply:
Unfortunately the smoke tests [still fail](https://pipelines.actions.githubusercontent.com/serviceHosts/54f2491d-97c6-4f3a-aa7c-ee03645d0508/_apis/pipelines/1/runs/14373/signedlogcontent/54?urlExpires=2023-01-16T10%3A22%3A47.8447431Z&urlSigningMethod=HMACV1&urlSignature=O1qsHqk9%2FLeIhtjy5qzXoc9RZqWgyIAMDIFkPLgx364%3D)when using the v2 SDK. I've opened a [new issue](https://github.com/microsoft/hi-ml/issues/733) for this specifically though, as it would be a great goal to aim for to demonstrate the v2 SDK being properly integrated.